src/home_robot_hw/home_robot_hw/remote/api.py

```diff
@@ -153,3 +157,34 @@ def dpt_cam(self):
     def get_frame_pose(self, frame, base_frame=None, lookup_time=None):
         """look up a particular frame in base coords"""
         return self._ros_client.get_frame_pose(frame, base_frame, lookup_time)
+
+    def go_to_manip_mode(self):
```

Comment:
The naming of functions is confusing for me. What is the difference between `switch_to_manipulation_mode` and `go_to_manip_mode`? It will be great if we can make the names more descriptive?

Reply:
calling them move_to_manip_posture

Reply:
Thanks